src/entity/task/vdaf/tests/serde.rs

```diff
@@ -0,0 +1,371 @@
+use serde_test::{assert_tokens, Token};
```

Comment:
Is it a hard requirement to use serde_test here as compared to comparing to string representations or serde_json::Value? I find this entire file very difficult to read (and I feel likewise about the usage in janus when I needed to modify it)

Reply:
I can take it out. Since we're only using `serde_json` in divviup-api, and not also `serde_yaml`, the JSON tests should be sufficient. I wasn't sure at first if I was going to replace newtype enum variants with struct enum variants, for instance.